templates/includes/incl--field--image-video-embed.html.twig

```diff
@@ -0,0 +1,54 @@
+{#
+/**
+ * @file
+ * An include to override image and video embed field templates.
+ * ~ Removed <div{{ attributes }}> from multiple section.
+ * ~ Removed all <div{{ item.attributes }}> wrappers.
+ * ~ Removed title_classes.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see field.html.twig
+ *
+ * @see template_preprocess_field()
+ *
+ * TO DO: get this (and the updated field--image and the new field--video-embed-field) into cwd_base and then remove all three from cwd_biotech!
+ */
+#}
+
+{% if label_hidden %}
+  {% if multiple %}
+    {% for item in items %}
+      {{ item.content }}
+    {% endfor %}
+  {% else %}
+    {% for item in items %}
+      {{ item.content }}
+    {% endfor %}
+  {% endif %}
+{% else %}
+  <div{{ attributes }}>
+    <div{{ title_attributes }}>{{ label }}</div>
+    {% if multiple %}
+      <div>
+    {% endif %}
```

Comment:
Do we want to just get rid of these divs, too? Seems like they are extraneous.

Reply:
I'm thinking keeping them for now, in case they're relied-upon for stuff -- certainly in favor of checking it out / testing, but with the other changes, there are no actual markup changes, ergo, minimal-to-no risk of unintended consequences.  So, I'd rather do that separately I think -- unless you disagree!  Thank you!

Reply:
Sure, I don't want to have to spend the time testing on that right now.